TTS/tts/datasets/TTSDataset.py

```diff
@@ -70,7 +72,7 @@ def __init__(self,
         self.sort_items()
 
     def load_wav(self, filename):
-        audio = self.ap.load_wav(filename)
+        audio = self.ap.load_wav(filename, sr=self.sample_rate)
```

Comment:
Let's do not resample audio here. It slows down the data loading a lot. User should resample audio in advance for better performance.

Reply:
OK!  I fixed this

Reply:
> @Edresson Nice results so far!
> 
> One question though, any findings on your side how robust the scaling of tokens or providing a ref. wav is regarding the attention? With ddc also new masking was introduced [L130](https://github.com/Edresson/TTS/blob/e20fac2ed34e7f21e54b8d0f1668f2d3a50b30ec/TTS/tts/models/tacotron2.py#L130) and [L136](https://github.com/Edresson/TTS/blob/e20fac2ed34e7f21e54b8d0f1668f2d3a50b30ec/TTS/tts/models/tacotron2.py#L136).
> In my previous tests those maskings would mostly break the attention when scaling a token.
> I've since retrained without the masking and the model seems more robust + flexible when conditioning on style tokens.
> 
> At last I have to say that using ddc in combination with gst has produces so far the best quality + robustness of model I've trained so far.

 My results with GST, do not allow many modifications. As Embeddings speakers are not good enough if I keep a lot of GST tokens, GST starts to learn the characteristics of the speaker. This way, GST did not allow generalizing to new speakers the only way I found was to drastically reduce the number of GST tokens (I used only 5 tokens). So my GST control results are not good. I believe that if we train a speaker encoder with much more data (Maybe using LibriTTS, VoxCeleb1 and VoxCeleb2), we can have a more accurate representation of each speaker and train a multispeaker model with a more robust GST.

In my tests scaling of tokens the model is more robust, when we use a .wav reference, depending on the sample the speaker speaks very fast, or even does not speak the first word!

Thanks for the tips :). It's good to know what worked best for you